eox_nelp/pearson_vue/rti_backend.py

```diff
@@ -0,0 +1,49 @@
+"""
+This module provides the RealTimeImport class, which is responsible for orchestrating the RTI pipeline
+and executing various processes related to rti.
+
+Classes:
+    RealTimeImport: Class for managing RTI operations and executing the pipeline.
+"""
+
+
+class RealTimeImport:
+    """
+    Class for managing RTI (Real Time Import) operations and executing the pipeline.
+
+    Attributes:
+        backend_data (dict): A dictionary containing backend-specific data.
+
+    Methods:
+        run_pipeline(): Executes the RTI pipeline by iterating through the pipeline functions.
+        get_pipeline(): Returns the RTI pipeline, which is a list of functions to be executed.
+    """
+
+    def __init__(self, **kwargs):
+        """
+        Initializes the RealTimeImport instance with the provided keyword arguments.
+
+        Args:
+            **kwargs: Additional keyword arguments to configure the RealTimeImport instance.
+        """
+        self.backend_data = kwargs.copy()
+
+    def run_pipeline(self):
+        """
+        Executes the RTI pipeline by iterating through the pipeline functions.
+        """
+        pipeline = self.get_pipeline()
+        pipeline_index = self.backend_data.get("pipeline_index", 0)
```

Comment:
why pipeline_index was selected to be a member of backend_data kwargs, instead of being a class attribute keep it as int ?
`self.pipeline_index`

Reply:
This could isolate the pipeline_index if some method change it forcing in the kwargs output of a ppeline_step.
eg:
 ``` python
def crazy_pipeline_step:
    return {"pipeline_index": 1000}
```

Reply:
When the task fails the retry method requires all this information, so it's easier to pass just one argument